src/DotNetty.Common/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Nito.Deque" version="1.0.0" targetFramework="net45" />
```

Comment:
Remove this line from the packages.config file or adjust the FAKE script so it doesn't accidentally include Nito as a dependency in our own NuGet package then (since FAKE builds the NuSpec assuming that everything in packages.config for this project is a dependency)